src/main/java/uk/gov/companieshouse/api/accounts/Kind.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.companieshouse.api.accounts;
+
+public enum Kind {
+
+    ACCOUNT("accounts"),
+    ABRIDGED_ACCOUNT("accounts#abridged"),
```

Comment:
Should we be supporting abridged accounts at this time?

Reply:
Nothing wrong with putting it in, besides, this may go with Steves code after the refactor

Reply:
I think it says that we support a certain type of accounts in this service when in reality we don't

Reply:
removed